indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -426,6 +426,11 @@ def can_access(self, user):
             return True
         return self.can_review(user)
 
+    def can_comment(self, user):
+        if not user:
+            return False
+        return self.can_judge(user) or self.can_convene(user) or self.can_review(user)
```

Comment:
Yes, except that it's more readable since it's split in access checks and "not even logged in"